scipy-1.0/paper.tex

```diff
@@ -122,6 +122,21 @@ \section*{Project organisation and community}
 
 \textbf{Governance}
 
+SciPy adopted an official Governance Document on August 3, 2017 (see PR\#6953). In summary,
+we have a Benevolant Dictator for Life (BDFL) in Pauli Virtanen, and a (currently) 18
+member Steering Council chaired by Ralf Gommers that i.e., decides on who gets commit rights.
```

Comment:
i.e. means "that is"
I think this should be e.g. ("for example") but better I think would be to state broadly what the steering council does (even if it amounts to nothing more than a rewording of "steers the development of SciPy") and then give examples, if desired.